build/images/ovs/apply-patches.sh

```diff
@@ -35,48 +35,50 @@ function version_let() { test "$(printf '%s\n' "$@" | sort -V | head -n 1)" == "
 # greater than or equal to
 function version_get() { test "$(printf '%s\n' "$@" | sort -rV | head -n 1)" == "$1"; }
 
-if version_lt "$OVS_VERSION" "2.13.0" || version_gt "$OVS_VERSION" "2.13.1"; then
-    echoerr "OVS_VERSION $OVS_VERSION is not supported (must be >= 2.13.0 and <= 2.13.1)"
+if version_lt "$OVS_VERSION" "2.13.0" || version_gt "$OVS_VERSION" "2.14.0"; then
+    echoerr "OVS_VERSION $OVS_VERSION is not supported (must be >= 2.13.0 and <= 2.14.0)"
     exit 1
 fi
 
 # We cannot use 3-way merge unless we are in a git repository. If we need 3-way
 # merge, we will need to clone the repository with git instead of downloading a
 # release tarball (see Dockerfile).
 
-# These 2 patches (post 2.13.x) ensures that datapath flows are not deleted on
-# ovs-vswitchd exit by default. Antrea relies on this to support hitless upgrade
-# of the Agent DaemonSet.
-# The second patch depends on the first one.
-curl https://github.com/openvswitch/ovs/commit/586cd3101e7fda54d14fb5bf12d847f35d968627.patch | \
-    git apply
-# We exclude 2 files which are likely to cause conflicts.
-curl https://github.com/openvswitch/ovs/commit/79eadafeb1b47a3871cb792aa972f6e4d89d1a0b.patch | \
-    git apply --exclude NEWS --exclude vswitchd/ovs-vswitchd.8.in
-
-# This patch (post 2.13.x) ensures that ovs-vswitchd does not delete datapath
-# ports on exit.
-curl https://github.com/openvswitch/ovs/commit/7cc77b301f80a63cd4893198d82be0eef303f731.patch | \
-    git apply
-
 # This patch (post 2.13.0) ensures that ct_nw_src/ct_nw_dst supports IP Mask.
 if version_let "$OVS_VERSION" "2.13.0"; then
     curl https://github.com/openvswitch/ovs/commit/1740aaf49dad6f533705dc3dce8d955a1840052a.patch | \
         git apply
 fi
 
-# These patches (post 2.13.x) are needed to fix the debian build on Ubuntu 20.04.
-curl https://github.com/openvswitch/ovs/commit/c101cd4171cfe04e214f858b4bbe089e56f13f9b.patch | \
-    git apply
-curl https://github.com/openvswitch/ovs/commit/3c18bb0fe9f23308061217f72e2245f0e311b20b.patch | \
-    git apply
-curl https://github.com/openvswitch/ovs/commit/fe175ac17352ceb2dbc9958112b4b1bc114d82f0.patch | \
-    git apply
+if version_get "$OVS_VERSION" "2.13.0" && version_let "$OVS_VERSION" "2.13.1" ; then
```

Comment:
I think the test should just be `if version_lt "$OVS_VERSION" "2.14.0"` here

Reply:
Good idea. Updated.